conf/messages

```diff
@@ -493,6 +493,60 @@ otherBeneficiary.whenRemoved.error.invalid = Enter a real date when the benefici
 otherBeneficiary.whenRemoved.error.future = The date the beneficiary was removed from the trust must be today or in the past
 otherBeneficiary.whenRemoved.error.past = The date the beneficiary was removed must be on or after the date they were added
 
+otherBeneficiary.description.title = What is the description of the other type of beneficiary?
+otherBeneficiary.description.heading = What is the description of the other type of beneficiary?
+otherBeneficiary.description.checkYourAnswersLabel = What is the description of the other type of beneficiary?
+otherBeneficiary.description.hint = This must be 56 characters or less. For example, future grandchildren and future great-grandchildren.
```

Comment:
Prototype reads "Do not include named or specific individuals, they must be listed as individual beneficiaries. This must be 70 characters or less". If this is correct it will require a new form provider that points to the correct validation regex

Reply:
Yep schema says
`"^[0-9a-zA-Z{\u00C0-\u02FF\u2019}\\- \\u005C&`'^]{1,70}$"`

Reply:
I've done this now but other than the length it seems to need the same validation as name so I've used the nameRegex instead of descriptionRegex as that seems to do something different?